### PR TITLE
Josepharaoh's big PR

### PR DESCRIPTION
**Pull request by tamashihoshi**
see:
https://forum.tuxemon.org/thread.php?pid=998#p998

### CODE REVIEW

**Comment by tamashihoshi** [comment]
For credits, look here:
https://forum.tuxemon.org/thread.php?pid=1014#p1014

File: tuxemon/resources/maps/map1_apartment2.tmx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  <image source="../gfx/tilesets/stairs.png" width="128" height="48"/>
+ </tileset>
+ <tileset firstgid="217" name="KelvinShadewing_Buildings" tilewidth="16" tileheight="16" tilecount="500">
+  <image source="../gfx/tilesets/KelvinShadewing_Buildings.png" width="320" height="400"/>
+ </tileset>
+ <tileset firstgid="717" name="setPiecesTSR" tilewidth="16" tileheight="16" tilecount="1312">
+  <image source="../gfx/tilesets/setPiecesTSR.png" width="671" height="512"/>
+ </tileset>
+ <tileset firstgid="2029" name="floorsandwalls" tilewidth="16" tileheight="16" tilecount="88">
+  <image source="../../../../../Users/Judy/Downloads/tuxemon-unstable-latest/tuxemon-unstable-latest/tuxemon/resources/gfx/tilesets/floorsandwalls.png" width="176" height="128"/>
```
Comment (ShadowApex) [inline tuxemon/resources/maps/map1_apartment2.tmx]: Thanks tamashihoshi for adapting Josepharaoh's changes to a pull request. Before I can accept these changes, there are a few references like this to a path that does not exist on everyone's machine. All maps should refer to the *relative* path of tilesets in the resources directory. 

Example:
Doesn't exist for everyone: `../../../../../Users/Judy/Downloads/tuxemon-unstable-latest/tuxemon-unstable-latest/tuxemon/resources/gfx/tilesets/floorsandwalls.png`    
Does exist for everyone: `../gfx/tilesets/floorsandwalls.png`

**Comment by ShadowApex** [comment]
I'll also need some time to hunt down the attributions and add them to the CREDITS.md file to make sure credit is properly given for any new assets.

File: tuxemon/resources/maps/map1_apartment2.tmx (hunk 1 above)
Comment (tamashihoshi) [inline tuxemon/resources/maps/map1_apartment2.tmx]: I hope I didn't miss anything, but all wrong paths in the map files should be fixed now

**Pull request merged**